modules/apps/foundation/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/session.js

```diff
@@ -505,6 +505,7 @@ AUI.add(
 						instance._intervalId = host.registerInterval(
 							function(elapsed, interval, hasWarned, hasExpired, warningMoment, expirationMoment) {
 								if (!hasWarned) {
+									instance._host.extend();
```

Comment:
Could you explain when the method `instance._uiSetActivated();` is called? Is it redundant on line 509?

Reply:
I think the method `instance._uiSetActivated()` is called to close the warning panner. Let me check again. Thanks